static/app/views/settings/project/navigationConfiguration.tsx

```diff
@@ -64,6 +64,11 @@ export default function getConfiguration({
           title: t('User Feedback'),
           show: () => !isSelfHostedErrorsOnly,
         },
+        {
+          path: `${pathPrefix}/toolbar/`,
+          title: t('Developer Toolbar'),
+          show: () => true || !!organization?.features?.includes('dev-toolbar-ui'),
```

Comment:
TODO: don't ship the `true ||` part when the time comes.

Reply:
```suggestion
          show: () => organization?.features?.includes('dev-toolbar-ui'),
```

Reply:
Good catch 😅